src/pubtools/sign/signers/msgsigner.py

```diff
@@ -431,9 +420,25 @@ def container_sign(self: MsgSigner, operation: ContainerSignOperation) -> Signin
             retries=self.retries,
             errors=errors,
         )
-        recvc.run()
-        errors = recvc.errors
+        recvt = RecvThread(recvc)
+        recvt.start()
 
+        errors = SendClient(
+            messages=messages,
+            broker_urls=self.messaging_brokers,
+            cert=self.messaging_cert_key,
+            ca_cert=self.messaging_ca_cert,
+            retries=self.retries,
+            errors=errors,
+        ).run()
+
+        if errors:
+            signer_results.status = "error"
+            for error in errors:
+                signer_results.error_message += f"{error.name} : {error.description}\n"
+            return signing_results
+
+        errors = recvc.errors
```

Comment:
Should be recvc._errors. And MessagingTimeout is not added to _errors by the time of this check.

Reply:
```suggestion
        errors = recvc._errors
```
RecvClient has _errors instead of errors. The fix is also needed for clear sign.